kuka_quantec_support/config/kr120_r2700_2_joint_limits.yaml

```
@@ -0,0 +1,39 @@
+default_velocity_scaling_factor: 1.0
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)